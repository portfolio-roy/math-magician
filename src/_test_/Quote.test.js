import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quotes';

it('renders when there is quote', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
