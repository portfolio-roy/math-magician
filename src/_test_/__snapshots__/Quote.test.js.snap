// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is quote 1`] = `
<div
  className="container"
>
  <p
    className="quote"
  >
    <blockquote>
      "Mathematics is the language with which God has written the universe."
    </blockquote>
     
    â€” Galileo Galilei
  </p>
</div>
`;
